### ISSUE TITLE
Static Linux GraalVM binary doesn't work

### ISSUE THREAD

**Issue by ericdallo** [clojure-lsp/clojure-lsp#296]
ATM the static Linux binary compiled with GraalVM doesn't work because of a non static sqlite lib described here: https://github.com/xerial/sqlite-jdbc/issues/413#issuecomment-772729676

**Comment by mageddo** [comment]
@ericdallo When you compile the binary using Graal you need to specify which files will be bundled within the binary file, something [like this][1] , take a look.

[1]: https://github.com/mageddo/graalvm-examples/blob/59f1f1bf09894681edfddaa100b4504770ad0685/sqlite/build.gradle#L28

**Comment by ericdallo** [comment]
Thanks @mageddo but we already do that here: https://github.com/clojure-lsp/clojure-lsp/blob/master/graalvm/native-unix-compile.sh#L29

**Comment by ericdallo** [comment]
https://github.com/xerial/sqlite-jdbc/issues/584

**Comment by zilti** [comment]
Not sure if it is due to me already having that dependency installed on my machine, but the static binary works just fine for me, system: openSUSE Tumbleweed.

**Comment by ericdallo** [comment]
Good to know @zilti, I'll double check it in a ubuntu

**Comment by kumarshantanu** [comment]
Ubuntu 20.04 AMD64, clojure-lsp `2021.03.18-19.23.41` static binary.

When I run `clojure-lsp` without any args, it just dies with a segmentation fault on Java 8, 11 and 16.

```
$ clojure-lsp
Segmentation fault (core dumped)
$ clojure-lsp --help
clojure-lsp 2021.03.18-19.23.41
clj-kondo 2021.03.04-SNAPSHOT
```

**Comment by ericdallo** [comment]
Yes, the issue is when clojure-lsp use sqlite lib that has no support for [static yet](https://github.com/clojure-lsp/clojure-lsp/issues/296#issuecomment-772931716)
